File: docs/components_page/components/dropdown/size.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

items <- list(
  dbcDropdownMenuItem("First"),
  dbcDropdownMenuItem(divider = TRUE),
  dbcDropdownMenuItem("Second")
)

dropdown <- htmlDiv(
  list(
    dbcDropdownMenu(
      label = "large dropdown",
      size = "lg",
      children = items,
      className = "mb-3"
    ),
    dbcDropdownMenu(
      label = "normal dropdown", children = items, className = "mb-3"
    ),
    dbcDropdownMenu(
      label = "small dropdown", size = "sm",
      children = items
    )
  )
)

File: docs/components_page/components/dropdown/size.jl
using DashBootstrapComponents, DashHtmlComponents

items = [
    dbc_dropdownmenuitem("First"),
    dbc_dropdownmenuitem(divider = true),
    dbc_dropdownmenuitem("Second"),
];

dropdown = html_div([
    dbc_dropdownmenu(
        label = "large dropdown",
        size = "lg",
        children = items,
        className = "mb-3",
    ),
    dbc_dropdownmenu(label = "normal dropdown", children = items, className = "mb-3"),
    dbc_dropdownmenu(label = "small dropdown", size = "sm", children = items),
]);
